test/avrora/storage/registry_test.exs

```diff
@@ -273,6 +273,20 @@ defmodule Avrora.Storage.RegistryTest do
       assert :ok == Registry.get(1) |> elem(0)
     end
 
+    test "when request should perform SSL verification based on given cert file using :ssl options" do
+      stub(Avrora.ConfigMock, :registry_ssl_opts, fn -> [verify: :verify_peer, cacertfile: "path/to/file"] end)
```

Comment:
Please change the test that it can show that new options take precedence over other old.